src/webAppCreator.ts

```diff
@@ -43,13 +43,13 @@ export class WebAppCreator extends WizardBase {
         return (<WebsiteStep>websiteStep).website;
     }
 
-    protected beforeExecute(step: WizardStep, stepIndex: number) {
+    protected beforeExecute(_step: WizardStep, stepIndex: number) {
```

Comment:
Can we just get rid of beforeExecute completely? This is the only place it's used and the output already has enough info:
```
Start creating new Web App...
The new Web App will be created in subscription "" ().
Existing resource group "" will be used.
Existing App Service Plan "" will be used.
Creating new Web App ""...
Web App "" created: 
```

Specifically, the 1st and 5th lines seem redundant to me

Reply:
@whiskeyjay I have no opinion on this-- Jay?